srcv2/components/Footer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import Section from './Section'
+
+import { GITHUB_LOGO_URL } from '../constants'
+
+const Footer = () => {
+    const renderSocialMedia = () => {
+        const socialMedia = [
+            {
```

Comment:
Should go in constants folder imo

Reply:
Resolved [d59c17f](https://github.com/setlife-network/trinary/pull/720/commits/d59c17f3d509f5034c9ca4919b9061be3623e993)